docs/workflows.md

```diff
@@ -0,0 +1,234 @@
+# Workflows
+
+Workflows are comprised of one or more AWS Lambda Functions and ECS Activities to discover, ingest, process, manage and archive data.
```

Comment:
Change to 'AWS Lambda Functions or ECS Activities'

Reply:
I think it should be `and` - you can have Lambda Functions and ECS activities in the same workflow